drupal_module_template.info.yml

```
@@ -2,5 +2,6 @@ name: OpenEuropa template
 description: OpenEuropa Drupal module template.
 package: OpenEuropa
 
+core_version_requirement: ^8 || ^9
```

I suppose it should be like that:
```suggestion
core_version_requirement: ^8.9 || ^9.1
```
also looks like we have to remove deprecated for 8.9 already key `core` according to this document: https://www.drupal.org/node/3070687.

---

Drone is failed because you need to remove the deprecated `core` key.